### PR TITLE
Couchbase - fail over node

### PR DESCRIPTION
**Pull request by mihaitusa**
Added op.
Signed off by Tusa Mihai: tusa.mihai@gmail.com

### CODE REVIEW

**Comment by mihaitusa** [comment]
😆

**Comment by mihaitusa** [comment]
@tethryus  ... seems that coverage/coveralls block this PR...

File: cs-couchbase/src/main/java/io/cloudslang/content/couchbase/entities/couchbase/NodesUri.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    private final String key;
+    private final String value;
+
+    NodesUri(String key, String value) {
+        this.key = key;
+        this.value = value;
+    }
+
+    public static String getNodesUri(String input) {
```
Comment (Bonczidai) [inline cs-couchbase/src/main/java/io/cloudslang/content/couchbase/entities/couchbase/NodesUri.java]: I'd call the input param `key` and the method `getValue()` -> `NodesUri.getValue(key)`'s responsability is clear to me at first sight

File: cs-couchbase/src/main/java/io/cloudslang/content/couchbase/factory/nodes/NodeHeadersBuilder.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import static io.cloudslang.content.couchbase.entities.constants.Constants.HttpClientInputsValues.FORM_URL_ENCODED;
+import static io.cloudslang.content.couchbase.entities.constants.Constants.NodeActions.FAIL_OVER_NODE;
+
+/**
+ * Created by TusaM
+ * 5/9/2017.
+ */
+public class NodeHeadersBuilder {
+    public NodeHeadersBuilder() {
+        // prevent instantiation
```
Comment (Bonczidai) [inline cs-couchbase/src/main/java/io/cloudslang/content/couchbase/factory/nodes/NodeHeadersBuilder.java]: shoudn't you prevent instantioation with a private constructor?

File: cs-couchbase/src/main/java/io/cloudslang/content/couchbase/factory/views/ViewHeadersBuilder.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ * Created by TusaM
+ * 4/28/2017.
+ */
+public class ViewHeadersBuilder {
+    private ViewHeadersBuilder() {
+        // prevent instantiation
+    }
+
+    public static void setViewHeaders(InputsWrapper wrapper) {
+        switch (wrapper.getCommonInputs().getAction()) {
```
Comment (Bonczidai) [inline cs-couchbase/src/main/java/io/cloudslang/content/couchbase/factory/views/ViewHeadersBuilder.java]: do we need the switch here?

Comment (mihaitusa) [inline cs-couchbase/src/main/java/io/cloudslang/content/couchbase/factory/views/ViewHeadersBuilder.java]: @Bonczidai  yes... for further ops

**Comment by mihaitusa** [comment]
@Bonczidai Merge it's up to you ... I've no longer access to do it....

**Comment by Bonczidai** [comment]
👌

**Pull request merged**